### DISCUSSION TITLE
API get blocked queries in the past minute

### DISCUSSION THREAD

**Discussion by ar3thien** [AdguardTeam/AdGuardHome#2992] [Q&A]
Hello,
I'm trying to build an API call that queries the DNS calls that were blocked in the past minute. is this something feasible? so far I have this call: http://adguard-hostip/control/querylog?response_status=blocked that returns some data. Is it possible to get only the data for the last minute? I saw there is an offset parameter, but I'm not sure I understand how it works.
Thanks

**Comment by ameshkov** [comment]
That's rather easy to do:
`/control/querylog?older_than=2021-04-21T22%3A25%3A27.370426251%2B03%3A00&limit=10000`

**Comment by ar3thien** [comment]
Hey @ameshkov 
thank you but I'm not sure that answers my need. let's say now is 2021-04-21T22:40:00+02:00
I want to get the queries from 2021-04-21T22:39:00+02:00 till now. so practically the queries that happened during the last minute. is that possible? so I need a call like newer_than=2021-04-21T22:39:00+02:00

**Reply by ameshkov** [reply]
Uh, my bad, I've misunderstood the question.

Nope, there're no parameters for that. The only way would be to query with `offset/limit` and read/re-request until you encounter a record for the previous minute.

**Reply by ar3thien** [reply]
Thank you, can you pls give me more details how the offset works? I didn't understand it from the API description.

**Reply by ameshkov** [reply]
`offset` and `limit` pair as usual
